tests/Http/HttpRequestTest.php

```diff
@@ -227,6 +227,12 @@ public function testHasMethod()
 
         $request = Request::create('/', 'GET', ['foo' => ['bar' => 'baz']]);
         $this->assertTrue($request->has('foo.bar'));
+
+        //tests objects from JSON post requests
+        $object = new stdClass();
```

Comment:
Why would this every be an object. It's only allowed to pass arrays to this method.